implicit-fallthrough.patch

When compiling using gcc-7 with -Werror=implicit-fallthrough enabled,
implicit case statement fallthroughs are not permitted.

This patch uses the gcc "comment method" to make the fallthrough
explicit.  Using the comment method is backwardly compatible with
earlier versions of gcc.  See The gcc documentaion for the
-Wimplicit-fallthrough=3 option.

diff --git a/src/authvar.c b/src/authvar.c
index ad659ca..0c5d752 100644
--- a/src/authvar.c
+++ b/src/authvar.c
@@ -511,6 +511,7 @@ main(int argc, char *argv[])
 	case IMPORT|SET:
 	case IMPORT|SIGN|SET:
 		fprintf(stderr, "authvar: not implemented\n");
+		/* fallthrough */
 	case IMPORT|SIGN|EXPORT:
 	default:
 		fprintf(stderr, "authvar: invalid flags: ");
